Source/WebKit/Platform/cocoa/ExtensionCapability.h

```diff
@@ -28,19 +28,22 @@
 #if ENABLE(EXTENSION_CAPABILITIES)
 
 #include <wtf/Forward.h>
+#include <wtf/RetainPtr.h>
 
-OBJC_CLASS _SECapability;
+OBJC_CLASS BEProcessCapability;
 
 namespace WebKit {
 
 class ExtensionCapability {
 public:
     virtual ~ExtensionCapability() = default;
     virtual String environmentIdentifier() const = 0;
-    virtual RetainPtr<_SECapability> platformCapability() const = 0;
+    BEProcessCapability *platformCapability() const { return m_platformCapability.get(); }
 
 protected:
     ExtensionCapability() = default;
+
+    RetainPtr<BEProcessCapability> m_platformCapability;
```

Comment:
This should be private. protected data members are not a good design.

Reply:
Ah, good point, fixed!